Specifications/QIR/Callables.md

```diff
@@ -1,9 +1,9 @@
-## Callables
+# Callables
 
 We use to term _callable_ to mean a subroutine in the source language.
 Different source languages use different names for this concept.
 
-### Basics
+## Basics
 
```

Comment:
Would be nice to see either a definition or a link explaining what exactly _functor_ means in this context. The meaning that I intuitively feel (but not necessarily correctly understand) here, differs from the meaning that I remember from the C++ books.

Reply:
Indeed functors here are a somewhat quantum specific concept. I am not sure there are good links to just add here, and I'll leave it for a separate PR to potentially elaborate and explain more.